src/docker-osx-dev

```diff
@@ -226,9 +225,9 @@ function do_rsync {
 #
 function sync {
   local readonly paths_to_sync=("$@")
-
-  for path in "${paths_to_sync[@]}"; do
-    do_rsync "$path"
+  for pathRsync in ${paths_to_sync[@]}
```

Comment:
Right, but I'm trying to understand why the quotes are doing this in this case, but not in all the others. I feel like there is a gap in my knowledge of bash arrays or loops, and as a result, there may be similar bugs lurking elsewhere.